packages/core/src/pattern.mjs

```diff
@@ -55,7 +56,7 @@ export function Pattern(designConfig) {
   __addNonEnumProp(this, '__hide', {})
 
   // Enumerable properties
-  this.designConfig = designConfig // The design configuration (unresolved)
+  this.designConfig = cloneDeep(designConfig) // The design configuration (unresolved)
```

Comment:
necessary because the design config was being shallow copied from the gist, so runtime changes that were supposed to be ephemeral were also mutating the gist, and then if a gist save was triggered those changes were polluting the gist